### PR TITLE
Convert numeric default values to integer

### PR DESCRIPTION
**Pull request by joshuaghezzi**
- Converting numeric default values to integer.

### CODE REVIEW

File: action/store/postgres.py [hunk 1]
```diff
@@ -118,5 +118,7 @@ def _clean_column_default(value: str) -> Any:
             value = value[0 : -len("::text")]
             if value[0] == value[-1] == "'" and value.count("'") == 2:
                 value = value[1:-1]
+        if value.isnumeric():
```
Comment (gouline) [inline action/store/postgres.py]: Do we have any booleans? Would be nice to include them too

Comment (joshuaghezzi) [reply action/store/postgres.py]: Yeah, we do. Added.

File: action/store/postgres.py [hunk 2]
```diff
@@ -117,6 +117,13 @@ def _clean_column_default(value: str) -> Any:
...
-
-        return value
+                return value[1:-1]
+        elif value.isnumeric():
+            return int(value)
+        elif value.lower() in (
+            "true",
+            "false",
+        ):
+            return bool(value)
```
Comment (gouline) [inline action/store/postgres.py]: Try `bool("FALSE")` in an interactive interpreter and prepare to be shocked

Comment (joshuaghezzi) [reply action/store/postgres.py]: Whoops 😅

File: action/store/postgres.py [hunk 3]
```diff
@@ -117,6 +117,12 @@ def _clean_column_default(value: str) -> Any:
...
-        return value
+                return value[1:-1]
+        elif value.isnumeric():
+            return int(value)
+        elif value.lower() == "true":
+            return True
+        elif value.lower() == "false":
+            return False
+        else:
+            return value
```
Comment (gouline) [inline action/store/postgres.py]:
```suggestion

        return value
```

Redundant trailing `else`

**Review by gouline — APPROVED** [review]

**Pull request merged**